extensions/applicationinsights-properties-js/src/Context/Session.ts

```diff
@@ -51,18 +51,33 @@ export class _SessionManager {
 
     public automaticSession: Session;
 
-    constructor(config: ISessionConfig, core?: IAppInsightsCore) {
+    constructor(config: ISessionConfig, core?: IAppInsightsCore, unloadHookContainer?: IUnloadHookContainer) {
         let self = this;
+        let _storageNamePrefix: string;
         let _cookieUpdatedTimestamp: number;
         let _logger: IDiagnosticLogger = safeGetLogger(core);
         let _cookieManager: ICookieMgr = safeGetCookieMgr(core);
+        let _sessionExpirationMs: number;
+        let _sessionRenewalMs: number;
 
         dynamicProto(_SessionManager, self, (_self) => {
    
             if (!config) {
                 config = ({} as any);
             }
     
+            let unloadHook = onConfigChange(config, (details) => {
+                _sessionExpirationMs = config.sessionExpirationMs || ACQUISITION_SPAN;
```

Comment:
Just converting these back to local variables to avoid function calls on every usage (this was what the TODO was about)